root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx

```diff
@@ -215,28 +218,96 @@
     <value>Update、Deleteクエリの検索条件に指定するタイムスタンプ
 を必須にする場合は、右のチェック ボックスをオンに設定する</value>
   </data>
-  <data name="label5.Text" xml:space="preserve">
-    <value>LIKE記号：</value>
+  <data name="cbxTSIndisp.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTSIndisp.Text" xml:space="preserve">
+    <value>タイムスタンプ必須</value>
+  </data>
+  <data name="cbxTypedDataSet.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTypedDataSet.Text" xml:space="preserve">
+    <value>型付きデータセット</value>
+  </data>
+  <data name="gbxDTO.Text" xml:space="preserve">
+    <value>DTOを生成する（エンコーディングはDaoと同じになる</value>
+  </data>
+  <data name="lblEscapeChar.Size" type="System.Drawing.Size, System.Drawing">
+    <value>83, 13</value>
   </data>
   <data name="lblEscapeChar.Text" xml:space="preserve">
     <value>エスケープ文字：</value>
   </data>
-  <data name="lblFamilyName.Text" xml:space="preserve">
-    <value>姓：</value>
+  <data name="chkEscapeToNChar.Size" type="System.Drawing.Size, System.Drawing">
+    <value>164, 17</value>
   </data>
-  <data name="lblSetDaoDefinition.Text" xml:space="preserve">
-    <value>D層定義情報ファイル</value>
+  <data name="chkEscapeToNChar.Text" xml:space="preserve">
+    <value>エスケープ文字をTO_NCHAR</value>
```

Comment:
@yusukemaegawa 

This can be understood by reference to the code.
Similarly,  if search, it is easy to find.
https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Frameworks/Tools/DaoGen_Tool/Form2.cs#L3787